vignettes/SKG2020_submission/skg2020_submission.Rmd

```diff
@@ -72,24 +80,37 @@ So far, there are four parsers available:
 
 These parsers can be used alone, or together like this:
 
-```r
-library(xml2) # working with xml files
-library(future) # parallel computing
-library(future.apply) # functional programming in parallel 
+```{r include=FALSE}
 openaire_records <- list.files("data/records", full.names = TRUE)
-future::plan(multisession) # initalize parallel computing
-oaire_data <- future.apply::future_lapply(openaire_records,
- function(files) {
-  # load xml file
-  doc <- xml2::read_xml(files)
-  # parser
-  out <- oarg_publications_md(doc)
-  out$linked_projects <- list(oarg_linked_projects(doc))
-  out$linked_ftxt <- list(oarg_linked_ftxt(doc))
-  # use file path as id
-  out$id <- files
-  out
-})
```

Comment:
unfortunately, this currently doesn't work on the CI as per #12 so I had to build a conditional around it